src/kpro_sup.erl

```diff
@@ -0,0 +1,13 @@
+-module(kpro_sup).
+
+-behaviour(supervisor).
+
+-export([start_link/0, init/1]).
+
+start_link() -> supervisor:start_link({local, ?MODULE}, ?MODULE, []).
```

Comment:
If you do this you also need to add `?MODULE` to the `registered` list in `.app.src`.

Reply:
You are right, its useless to register an unused supervisor, will remove the registration. Thx